apps/web/src/server/applications/common/components/form/form-case.component.js

```diff
@@ -31,6 +34,52 @@ export async function caseNameAndDescriptionData(request, locals) {
 	return { values: { title, description } };
 }
 
+/**
+ * Format stage property for stage edit page
+ *
+ * @param {import('express').Request} request
+ * @param {Record<string, any>} locals
+ * @returns {Promise<ApplicationsCreateCaseStageProps>}
+ * */
+export async function caseStageData(request, locals) {
+	/** @type {{name: string, displayNameEn: string}[]} */
+	const allStages = await getAllCaseStages();
+	const stages = allStages.filter((stage) => stage.name !== 'draft');
```

Comment:
allStages could be undefined or an error object from the api. consider using a "?" or a `allStage||[]`

Reply:
👍 That's done